lms/templates/courseware/proctored-exam-status.underscore

```diff
@@ -7,13 +7,14 @@
         }
     %>
     <div class='exam-text'>
+        <% // xss-lint: disable=underscore-not-escaped %>
```

Comment:
What is this? Is it a comment?

Reply:
Yeah, comparable to an `eslint-disable-next-line` but targetting a different linting tool.